Content.IntegrationTests/Tests/ContrabandTest.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
+using Content.Shared.Contraband;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests;
+
+[TestFixture]
+public sealed class ContrabandTest
+{
+    [Test]
+    public async Task EntityShowDepartmentsAndJobs()
+    {
+        await using var pair = await PoolManager.GetServerClient();
+        var client = pair.Client;
+        var protoMan = client.ResolveDependency<IPrototypeManager>();
+        var componentFactory = client.ResolveDependency<IComponentFactory>();
+
+        await client.WaitAssertion(() =>
+        {
+            var protos = protoMan.EnumeratePrototypes<EntityPrototype>()
+                .Where(p => !p.Abstract)
+                .Where(p => !pair.IsTestPrototype(p))
+                .Where(p => p.TryGetComponent<ContrabandComponent>(out _, componentFactory))
```

Comment:
```suggestion
                .Where(p => p.HasComponent<ContrabandComponent>())
```
No need to get the component since we don't use it.